app/connector/mongodb/src/main/resources/META-INF/syndesis/connector/mongodb3.json

```diff
@@ -13,10 +13,9 @@
           "io.syndesis.connector.mongo.MongoProducerFindCustomizer"
         ],
         "inputDataShape": {
-          "name": "Criteria",
-          "description": "Text criteria",
-          "kind": "java",
-          "type": "java.lang.String"
+          "name": "Filter parameters",
+          "description": "Flat list of parameters contained in the filter provided by user",
```

Comment:
This is a bit confusing for me but I'm not a Mongo expert, would a Mongo user understand what's requested of them?